spec/spec_helper.rb

```diff
@@ -14,13 +14,6 @@
 require 'vcr'
 require 'cgi'
 
-# FIXME: This bypasses the lookup of Javascript dependencies in tests until they can be found and uncoupled
-class << ActionController::Base.helpers
-  def image_path(path, options = {})
-    path
-  end
-end
-
```

Comment:
For the record, I ran the UI classic test suite (not the javascript one though, just the specs), with out this in place with my local copy of `manageiq`, and all the tests passed (except one automate one that failed with master and my changes, so I didn't bother with counting that).